src/NetEscapades.AspNetCore.SecurityHeaders/Headers/PermissionsPolicy/FederatedLearningOfCohortsCalculationPermissionsPolicyDirectiveBuilder.cs

```diff
@@ -0,0 +1,17 @@
+﻿namespace NetEscapades.AspNetCore.SecurityHeaders.Headers.PermissionsPolicy
+{
+    /// <summary>
+    /// Controls whether the current document is allowed to use
```

Comment:
nit: missing a bit of the description
```suggestion
    /// Controls whether the current document is allowed to make a FLoC calculation
```